SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestMultipleOwnership/MultipleOwnershipPawn.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+
+#include "MultipleOwnershipPawn.h"
+
+#include "Components/StaticMeshComponent.h"
+#include "Components/SphereComponent.h"
+#include "Materials/Material.h"
+#include "Net/UnrealNetwork.h"
+
+AMultipleOwnershipPawn::AMultipleOwnershipPawn()
+{
+	bReplicates = true;
+#if ENGINE_MINOR_VERSION < 24
+	bReplicateMovement = true;
+#else
+	SetReplicatingMovement(true);
+#endif
+	CollisionComponent = CreateDefaultSubobject<USphereComponent>(TEXT("CollisionComponent"));
+	CollisionComponent->InitSphereRadius(15.0f);
```

Comment:
We can probably get by with just a `SceneComponent` since collision isn't used in the test.